packages/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -21,15 +36,76 @@
 	/** @type {string} */
 	export let name;
 
+	export let multiple = false;
+
+	export let hideDuringPrint = true;
+
 	/** @type {string} */
-	export let defaultValue = null;
+	export let defaultValue = undefined;
 
 	setContext('dropdown_context', {
-		defaultValue: defaultValue,
+		defaultValue,
 		hasBeenSet: false,
-		setSelectedValue: (selected) => ($inputs[name] = selected)
+		handleSelect,
+		multiple
 	});
 
+	const selectedValues = writable([]);
+	setContext('dropdown_selected_values', selectedValues);
+
+	function jsToDuckDB(value) {
+		if (value == null) return 'null';
+		if (typeof value === 'string') return `'${value.replaceAll("'", "''")}'`;
+		if (typeof value === 'number' || typeof value === 'bigint' || typeof value === 'boolean')
+			return String(value);
+		if (value instanceof Date) return `'${value.toISOString()}'::TIMESTAMP_MS`;
+		if (Array.isArray(value)) return `[${value.map((x) => jsToDuckDB(x)).join(',')}]`;
+		return JSON.stringify(value);
+	}
+
+	function selectedValuesToInput() {
+		let values;
+		if (multiple) {
+			values = $selectedValues.map((x) => x.value);
+			values.toString = () => jsToDuckDB(values);
+		} else {
+			values = $selectedValues[0]?.value ?? null;
+			// the default `toString` method for Dates aren't good for duckdb
+			if (values instanceof Date) values.toString = () => jsToDuckDB(values);
+		}
+
+		$inputs[name] = {
```

Comment:
This seems to deprecate support for `'${inputs.my_input_name}'` in favour of `'${inputs.my_input_name.label}'`

This is a breaking change

Reply:
hmm I'm not sure. 

I guess an alternative would be some kind of proxy that maps `inputs.my_input_name` -> `inputs.my_input_name.label`?

{}

Reply:
That would require proxying the string `inputs.my_input_name.label`, but you can't proxy strings (only objects)

One thing that's possible would be doing ```js
$inputs[name] = {
    // ...
    valueOf: () => label,
    toString: () => label
}
```
which would fix it in `SELECT * FROM somewhere WHERE id = '${inputs.something}'` and `inputs.something == 'blah'`, but *not* `inputs.something === 'blah'`

Reply:
<img width="235" alt="image" src="https://github.com/evidence-dev/evidence/assets/61282104/6bddea72-93ab-4d33-9e83-9cd92245350a">

Reply:
That seems reasonable to me

perhaps we can print a warning to the terminal if you use either that it is deprecated and should be replaced with `inputs.something.label`